modules/internal/ChapelArray.chpl

```diff
@@ -2919,7 +2988,10 @@ module ChapelArray {
       // intelligent will likely be needed before it is worth it to
       // parallelize whole-domain assignment for associative arrays.
       //
-      compilerWarning("whole-domain assignment has been serialized (see note in $CHPL_HOME/STATUS)");
+
+//      disabled for testing for the same reason
+//      as the array version: it can be called from autoCopy/initCopy.
+//      compilerWarning("whole-domain assignment has been serialized (see note in $CHPL_HOME/STATUS)");
```

Comment:
I'd like to keep it here but could re-write the comment. There is still a TO-DO here and I want some comment to indicate that.